src/sysinfo/nvapi_adapter.cpp

```diff
@@ -211,27 +228,38 @@ namespace dxvk {
         if (m_vkProperties.deviceID >= 0x2600)
             return NV_GPU_ARCHITECTURE_AD100;
 
+        // See https://gitlab.freedesktop.org/mesa/mesa/-/blob/main/src/nouveau/vulkan/nvk_physical_device.c
+        // for NVK properties and features
+
         // KHR_fragment_shading_rate's
-        // primitiveFragmentShadingRateWithMultipleViewports is supported on
-        // Ampere and newer
+        // primitiveFragmentShadingRateWithMultipleViewports is supported on Ampere and newer
+        // TODO: We haven't found a way yet to identify Ampere on NVK
         if (IsVkDeviceExtensionSupported(VK_KHR_FRAGMENT_SHADING_RATE_EXTENSION_NAME)
```

Comment:
Perhaps we should add `HasNvProprietaryDriver() &&` here for now?

Reply:
Yeah i was thinking about this too. My hopes were, that once NVK gets that extension, it will have the same architecture requirements as on the NV prop driver, so that we just have to remove the TODO at some point. But that might be indeed too optimistic.

Reply:
Done.